doc/report.Rmd

```diff
@@ -8,3 +8,19 @@ author:
 output: html_document
 ---
 
+
+Predator-prey relationships are ubiquitous throughout the world.  
+
+
+The natural environment is full of interactions between organisms.  These interactions can either be beneficial like mutualism such as plant producing nectar for ants which defend them or detrimental such as predator and prey.  
+
+
+Animals come in many different shapes and sizes and they can be seen interacting with each other in many forms.  One of the most common type of interaction is the predator-prey relationship which generates food webs.  In predator-prey interactions, we can see that 90% of predators are bigger than the prey that they eat (JOEL E. COHEN,1993).  In general, literature suggests that there is a correlation between prey mass and predator mass (Brose 2006).  There are a variety of factors that can affect predator-prey interactions.  From the Marine predator and prey body sizes dataset created by Barnes et al (2008), we determined that location and feeding behavior are just two of the aforementioned factors that could influence this interaction.
```

Comment:
Good start :) 

Some suggestions:
- I don't think we need to say that organism interactions or predator-prey relationships are common because that's sort of obvious. It might be more meaningful (in terms of our project) to say what size relationships in predator-prey interactions implicate in food web, ecosystem structure, trophic levels etc.
- I also don't think we need to mention the thing about the ants -- I get that it shows the variability of interactions but it doesn't really relate to our study since we're only considering predator-prey interactions. 
- The thing about 90% and the mass correlation is good. Studies were cited, maybe expand on what they found in terms of correlations in different kinds organisms, feeding types, environments etc (if any). Are there certain factors that are common across studies predicting size relationships? Any contradictions? Gaps in the research? What kind of theories are floating around? Set the stage some more here. 
- The last sentence could be a new paragraph about our project, but possibly explain what the dataset is and how it relates to predator prey mass correlations before saying anything about what we found in our analyses so far. 


Anyway, good stuff 👍